gammapy/spectrum/utils.py

```diff
@@ -13,19 +16,254 @@
            'np_to_pha',
            ]
 
-# TODO: remove functions now that we have classes!
 
+def find_log_centers(bin_edges):
+    """Compute energy bin log centers.
+
+    Bin center = sqrt(low_edge * high_edge)
+    """
+    return np.sqrt(bin_edges[:-1] * bin_edges[1:])
+
+
+def find_log_edges(bin_centers):
+    """Compute equally log-spaced energy bin edges
+    """
+
+    bin_edges = np.sqrt(bin_centers[:-1] * bin_centers[1:])
+    first = bin_centers[0] * bin_centers[0] / bin_edges[0]
+    last = bin_centers[-1] * bin_centers[-1] / bin_edges[-1]
+    bin_edges = np.append(first, bin_edges)
+    bin_edges = np.append(bin_edges, last)
+
+    return bin_edges
+
+
+class EnergyBinning(object):
```

Comment:
This class will go away, right?
I guess you're keeping it for now while you're moving functionality to the `Energy` and `EnergyBounds` classes?